meshapp/src/components/login-form.tsx

```diff
@@ -22,6 +25,52 @@ const LoginScreen = (props: ComponentProps) => {
     console.log(formData);
   };
 
+  const navigate = useNavigate();
+  const [cookies, setCookie] = useCookies(['user_id']);
+
+  useEffect(() =>{},[cookies]);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const response = await axiosInstance.post('accounts/login/',{
+      "email":formData.user,
+      "password":formData.pass
+    })
+      .then((axiosResponse) => {
+        if(axiosResponse.data.enabled_2fa){
+          setCookie('user_id', axiosResponse.data.user_id, { path:'/', maxAge:150 });
+          axiosInstance.post('accounts/set-two-factor-auth/', {
+            "accountID":axiosResponse.data.user_id
+          })
+          .then(() => {
+            navigate('/otp');
+          })
+          .catch((error) => {
+            alert(error.response.data.message);
+          });
+        }
+        else{
+          navigate('/logged_in_home')
+        }
+      })
+      .catch((error) => {
+        if(error.response){
```

Comment:
For error handling, it might be worth to use more descriptive erroring potentially included with the default error responses.

Reply:
I extended the error handler. But I wasn't sure if the error description was for the user or the console, so for now I only extended it for the console. [fc1474c](https://github.com/LetsMesh/Site/pull/316/commits/fc1474c655e43ab86f792d0326b90d3133981852)

Reply:
Yea it is for the console but currently it spits out the default exception. I think proper console logging / error logging would include a more "english" error statement before outputting the exception. This is something nice to have not necessarily required

Reply:
This is how it currently looks like for console error. https://github.com/LetsMesh/Site/blob/82aa9466cabd90755fe24d0b4a3da7e758675390/meshapp/src/config/errorHandlerModule.ts#L10-L33
Would that be fine? Also, I'd assume that the comments would be redundant here

Reply:
I think this is fine but I was more leaning towards specific error outputting. Like imagine putting in a password and instead of saying "incorrect password" the error message is just "oops something went wrong" or an exception stack strace. 

Specific error handling will tell you what you did wrong, even in console log so that if someone is debugging an exception they have clues. 

I like to add a specific english explanation for the specific error and then include stack trace afterwards.

Reply:
I made some changes. Let me know if anything is wrong. Also, I accidentally formatted login-form.tsx with prettier so it might've looked like many lines changed. c9c2d03